account_banking_payment_export/__openerp__.py

```diff
@@ -41,6 +41,7 @@
         'views/account_payment.xml',
         'views/payment_mode.xml',
         'views/payment_mode_type.xml',
+        'views/account_invoice.xml',
```

Comment:
I change it to allow to select the communication type in customer invoices (now it's a hidden field)